Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -748,7 +748,7 @@ void JitArm64::DoJit(u32 em_address, JitBlock* b, u32 nextPC)
 
       // Inline exception check
       LDR(IndexType::Unsigned, ARM64Reg::W30, PPC_REG, PPCSTATE_OFF(Exceptions));
-      TBZ(ARM64Reg::W30, 3, done_here);  // EXCEPTION_EXTERNAL_INT
+      TBZ(ARM64Reg::W30, IntLog2(EXCEPTION_EXTERNAL_INT), done_here);
```

Comment:
This doesn't seem equivalent? EXCEPTION_EXTERNAL_INT is 4 and log2(4) = 2 (and IntLog2 does seem to return 2), but the original code used 3. Is that an pre-existing bug?

Reply:
I would assume so. I haven't been able to observe any difference in behavior from changing it from 3 to 2, though.